From 177cea46a0f135a2d6de340d3f805c2bf91243b8 Mon Sep 17 00:00:00 2001
From: Robert Richter <rrichter@cavium.com>
Date: Thu, 12 Nov 2015 23:47:49 +0100
Subject: [PATCH 26/26] of/irq: Fix build error
Mime-version: 1.0
Content-type: text/plain; charset=UTF-8
Content-transfer-encoding: 8bit
Patch-mainline: Not yet, not sure if this is not an internal dependency

 drivers/of/irq.c: In function ‘of_msi_map_rid’:
 drivers/of/irq.c:605:60: error: dereferencing pointer to incomplete type
 drivers/of/irq.c:606:18: error: dereferencing pointer to incomplete type
 drivers/of/irq.c:609:39: error: dereferencing pointer to incomplete type
 drivers/of/irq.c:615:4: error: implicit declaration of function ‘dev_err’ [-Werror=implicit-function-declaration]
 drivers/of/irq.c:632:33: error: dereferencing pointer to incomplete type
 drivers/of/irq.c:656:2: error: implicit declaration of function ‘dev_dbg’ [-Werror=implicit-function-declaration]
 drivers/of/irq.c:658:3: error: implicit declaration of function ‘dev_name’ [-Werror=implicit-function-declaration]

Signed-off-by: Robert Richter <rrichter@cavium.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>

---
 drivers/of/irq.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/of/irq.c b/drivers/of/irq.c
index db44480..12fa7eb 100644
--- a/drivers/of/irq.c
+++ b/drivers/of/irq.c
@@ -18,6 +18,7 @@
  * driver.
  */
 
+#include <linux/device.h>
 #include <linux/errno.h>
 #include <linux/list.h>
 #include <linux/module.h>
-- 
2.1.4

